src/main/java/com/google/alpollo/ConfigHelper.java

```diff
@@ -0,0 +1,48 @@
+package com.google.alpollo;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonIOException;
+import com.google.gson.JsonSyntaxException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Objects;
+
+/**
+ * Helps to load configuration file and retrieve data from it.
+ */
+final public class ConfigHelper {
+  private static final String configFileName = "config.json";
```

Comment:
It is customary to write constant names in all caps, so this one should be something like CONFIG_FILE_NAME